aiocache/backends/redis.py

```diff
@@ -52,163 +39,171 @@ def __init__(
         port=6379,
         db=0,
         password=None,
-        pool_min_size=1,
-        pool_max_size=10,
+        pool_min_size=_NOT_SET,
+        pool_max_size=None,
         loop=None,
         create_connection_timeout=None,
         **kwargs,
     ):
         super().__init__(**kwargs)
+        if loop is not None:
+            warnings.warn(
+                "Parameter 'loop' has been obsolete on aiocache >= 0.12.0",
+                DeprecationWarning,
+            )
+        if pool_min_size is not _NOT_SET:
+            warnings.warn(
+                "Parameter 'pool_min_size' has been obsolete since aiocache >= 0.12.0",
+                DeprecationWarning,
+            )
+
         self.endpoint = endpoint
         self.port = int(port)
         self.db = int(db)
         self.password = password
-        self.pool_min_size = int(pool_min_size)
-        self.pool_max_size = int(pool_max_size)
+        # NOTE: In redis and aioredis 2.x, ConnectionPool raises ConnectionError
+        #  but doesn't wait for connection reuse when number of conns
+        #  exceeds the limit. While in aioredis 1.x, it waits asynchronously
+        #  via asyncio.Condition. So max pool size defaults to None in redis.
+        if pool_max_size is None:
+            self.pool_max_size = None
+        else:
+            self.pool_max_size = int(pool_max_size)
```

Comment:
```suggestion
        self.pool_max_size = None if pool_max_size is None else int(pool_max_size)
```

Reply:
We should probably just look at adding type annotations and then not doing this `int()` conversion...